library/objective-c/Envoy.mm

```diff
@@ -0,0 +1,45 @@
+#import "library/objective-c/Envoy.h"
+
+#import "library/common/main_interface.h"
+
+
+@interface Envoy ()
+@property (nonatomic, strong) NSThread *runner;
+@end
+
+@implementation Envoy
+
+@synthesize runner;
+
+// Create a new Envoy instance. The Envoy runner NSThread is started as part of instance
```

Comment:
These docstrings should probably live in the header file, right?

Reply:
Good call